spec.html

```diff
@@ -32006,7 +32006,7 @@
           1. Let _O_ be the *this* value.
           1. Perform ? ValidateTypedArray(_O_).
           1. Let _value_ be ? ToNumber(_value_).
-          1. Let _len_ be ? _O_.[[ArrayLength]].
+          1. Let _len_ be _O_.[[ArrayLength]].
```

Comment:
I agree this is concerning. Also agreed it's a separate issue. Removing the nonsense `?` for now :)